.github/workflows/release.yml

```diff
@@ -0,0 +1,49 @@
+name: Android Build and Publish
+
+permissions:
+  contents: write
```

Comment:
Why it needs write permissions?

Reply:
For writing(creating) a release.

See https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release